content/terraform/v1.12.x/docs/language/resources/configure.mdx

```diff
@@ -0,0 +1,142 @@
+
+> **Hands-on:** Try the [Terraform: Get Started](/terraform/tutorials/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorials.
+
+## Overview
+
+Complete the following steps to configure a resource:
+
+- Declare a resource and specify its type and a name.
+- Configure provider arguments. Most arguments in your resource configuration are specific to your provider.  
+- Configure Terraform meta-arguments arguments. Meta-arguments are arguments built into Terraform configuration language to control how Terraform creates and manages resources.
+
+## Prerequisites
+
+Choose and configure your provider. Providers are plugins for Terraform that offer a collection of resource types that you can provision. Refer to [Providers](/terraform/language/providers) for more information. 
+
+Most providers require authentication settings, version requirements, or other arguments to access their remote APIs. Terraform expects provider configuration in the root module. Refer to the [`provider` block reference](/terraform/language/block/provider) for details.
+
+To manage resources, a Terraform module must specify the required providers. Refer to [Provider requirements](/terraform/language/providers/requirements) for more information.
+
+## Declare a resource
+
+Add a `resource` block to your configuration, declare a type, and specify a name for it. The type determines the kind of infrastructure object the `resource` block manages and what arguments the resource supports. Most resource types are implemented by your provider, but Terraform includes a native resource type, [`terraform_data`](/terraform/language/resources/terraform-data), that you can use to create resources for specialized use cases. 
+
+You must also specify a local name for the resource. Terraform uses the resource type and name as the resource's address in the [state file](/terraform/language/state). 
+
+## Configure provider-specific arguments
+
+Most of the arguments within the body of a `resource` block are specific to the selected resource type as defined by the provider developer. Configure your resource by assigning values to the `resource` block’s arguments in your configuration arguments. The resource type's documentation lists which arguments are required and how to format their values. Refer to your provider documentation for details. 
+
+You can use Terraform [expressions](/terraform/language/expressions) and other dynamic Terraform language features to configure resource argument values in addition to hard-coding values.   
```

Comment:
```suggestion
You can use Terraform [Expressions](/terraform/language/expressions) and other dynamic Terraform language features to configure resource argument values in addition to hard-coding values.   
```

I *think* we match the page title, including case (other examples in this PR do it that way), but I can't find specific guidance on this in the style guide.